src/models/model.cpp

```diff
@@ -88,6 +92,9 @@ OrtValue* State::GetInput(const char* name) {
 }
 
 OrtValue* State::GetOutput(const char* name) {
+  if (OrtValue* output = extra_outputs_.GetOutput(name); output) {
```

Comment:
Is there a reason this line was written this way vs. the below?

```suggestion
  OrtValue* output = extra_outputs_.GetOutput(name);
  if (output) {
```

Reply:
applied.